quad_utils/config/filter_chain.yaml

```diff
@@ -78,7 +78,7 @@ grid_map_filters:
     type: gridMapFilters/MathExpressionFilter
     params:
       output_layer: traversability_raw
-      expression: 0.5 * (1.0 - (slope / 0.3)) + 0.5 * (1.0 - (roughness / 0.4))
+      expression: 0 * (1.0 - (slope / 0.3)) + 1.0 * (1.0 - (roughness / 0.4))
```

Comment:
Is this a temporary setup that makes us not use slopes to calculate traversability so we can walk on slopes?

Reply:
Yes, in the near future I would like to adjust local planner as well to check that grfs (either averaged over stance or at each index within stance) remain within the actual friction cone of the surface. Even in the short term though I don't think this change would break anything, it still pushes footholds away from steps and gaps via the roughness penalty.